langchain/src/chains/base.ts

```diff
@@ -83,6 +83,32 @@ export abstract class BaseChain<
     return this.call(input, config);
   }
 
+  private _validateOutputs(outputs: Record<string, unknown>): void {
```

Comment:
These changes don't seem related? Or were we missing them all along?

Reply:
Ported over from python so I'd assume we just were missing them to start with